pyproject.toml

```diff
@@ -14,7 +14,7 @@ python = ">=3.9,<3.12"
 astrapy = "~0.6.2"
 cassio = "~0.1.3"
 unstructured = "^0.10"
-llama-index = { version = "0.9.14", extras = ["langchain"] }
+llama-index = { version = "0.9.15.post2", extras = ["langchain"] }
```

Comment:
IIRC we said that we don't want to depend on .post releases. Maybe we can wait for the next train. There is no hurry